ol/miner/src/commit_proof.rs

```diff
@@ -18,10 +18,10 @@ pub fn commit_proof_tx(
     // Create a client object
     let mut client = LibraClient::new(tx_params.url.clone(), tx_params.waypoint).unwrap();
 
-    let chain_id = ChainId::new(client.get_metadata().unwrap().chain_id);
+    let chain_id = ChainId::new(client.get_metadata()?.chain_id);
```

Comment:
I think chain_id is already in TxParams, can you check?

Reply:
Closed